core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -1220,7 +1260,8 @@ export class ConversationSkinComponent {
         this.nextCard = nextCard;
         this.nextCardIfStuck = nextCardIfReallyStuck;
         if (!this._editorPreviewMode &&
-            !this.explorationPlayerStateService.isInQuestionMode()) {
+            this.explorationPlayerStateService.isPresentingIsolatedQuestions()
```

Comment:
If it's presenting isolated questions, then how can it be in preview mode?

Reply:
Updated